modules/meeting/app/services/recurring_meetings/ical_service.rb

```diff
@@ -105,6 +118,7 @@ def occurrence_event(schedule_start_time, meeting) # rubocop:disable Metrics/Abc
         e.dtend = ical_datetime meeting.end_time, tzid
         e.url = url_helpers.project_meeting_url(meeting.project, meeting)
         e.location = meeting.location.presence
+        e.sequence = 2
```

Comment:
According to the [specs](https://icalendar.org/iCalendar-RFC-5545/3-8-7-4-sequence-number.html) the sequence should be increased every time the calendar event is updated, right? We do have a `lock_version` on the meeting. Should we maybe use that as the sequence number?